samples/algorithms/database-search/Database Search.ipynb

```diff
@@ -53,9 +53,9 @@
    "outputs": [
     {
      "data": {
-      "application/json": "{\"name\":\"Microsoft.Quantum.Canon.ControlledOnInt\",\"kind\":1,\"source\":\"/home/jovyan/.nuget/packages/microsoft.quantum.standard/0.9.1908.2906/lib/netstandard2.1/Microsoft.Quantum.Standard.dll\",\"documentation\":\" # Summary\\n Returns a unitary operator that applies an oracle on the target register if the control register state corresponds to a specified positive integer.\\n\\n # Input\\n ## numberState\\n Positive integer.\\n ## oracle\\n Unitary operator.\\n\\n # Output\\n A unitary operator that applies `oracle` on the target register if the control register state corresponds to the number state `numberState`.\"}",
+      "application/json": "{\"name\":\"Microsoft.Quantum.Canon.ControlledOnInt\",\"kind\":1,\"source\":\"C:/Users/Chris/.nuget/packages/microsoft.quantum.standard/0.11.2003.3107/lib/netstandard2.1/Microsoft.Quantum.Standard.dll\",\"documentation\":\" # Summary\\n Returns a unitary operator that applies an oracle on the target register if the control register state corresponds to a specified positive integer.\\n\\n # Input\\n ## numberState\\n Positive integer.\\n ## oracle\\n Unitary operator.\\n\\n # Output\\n A unitary operator that applies `oracle` on the target register if the control register state corresponds to the number state `numberState`.\"}",
       "text/html": [
-       "<h4><i class=\"fa fas fa-terminal\"></i> Microsoft.Quantum.Canon.ControlledOnInt <a href=\"/home/jovyan/.nuget/packages/microsoft.quantum.standard/0.9.1908.2906/lib/netstandard2.1/Microsoft.Quantum.Standard.dll\"><i class=\"fa fas fa-code\"></i></a></h4><h1>Summary</h1>\n",
+       "<h4><i class=\"fa fas fa-terminal\"></i> Microsoft.Quantum.Canon.ControlledOnInt <a href=\"C:/Users/Chris/.nuget/packages/microsoft.quantum.standard/0.11.2003.3107/lib/netstandard2.1/Microsoft.Quantum.Standard.dll\"><i class=\"fa fas fa-code\"></i></a></h4><h1>Summary</h1>\n",
```

Comment:
Shouldn't we clean the outputs cells in this case?

Reply:
Agreed here, we shouldn't have per-user paths in the output. I ran locally this time, rather than in the devcontainer since the container still needed updated; hence the change in paths. I'll fix that back to `jovyan`. Thanks!